doc/api/async_context.md

```diff
@@ -462,11 +462,29 @@ changes:
   `AsyncResource`.
 * `thisArg` {any}
 
-Binds the given function to the current execution context.
+Binds the given function to the current execution context using a new
+`AsyncResource`.
 
 The returned function will have an `asyncResource` property referencing
 the `AsyncResource` to which the function is bound.
 
+### Static method: `AsyncResource.bindCurrent(fn[, thisArg])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `fn` {Function} The function to bind to the current execution context.
```

Comment:
When would one use `bindCurrent` instead of `bind` ?

Reply:
Almost any time you'd think you'd want to use the static `bind`. It avoids the creation of a new `AsyncResource` instance, which in many cases isn't actually needed.

Reply:
Should we explain in the documentation when `bind` is actually needed?